### PR TITLE
Pivotal ID # 178166470: Upgrade Kotlin and Kotlin corotuines

### PR DESCRIPTION
**Pull request by Juan-EBI**
https://www.pivotaltracker.com/story/show/178166470

### CODE REVIEW

File: commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/security/RegisterRequest.kt [hunk 1]
```diff
@@ -3,6 +3,7 @@ package ebi.ac.uk.api.security
 import com.fasterxml.jackson.annotation.JsonProperty
 import javax.validation.constraints.Email
 
+@Suppress("LongParameterList")
```
Comment (jhoanmanuelms) [inline commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/security/RegisterRequest.kt:6]: I see a lot of these were added during this refactor. Shouldn't we just adjust the parameter to something that makes more sense to us?

File: gradle.properties [hunk 2]
```diff
@@ -1 +1,2 @@
 org.gradle.parallel=false
+org.gradle.jvmargs=-Xmx4096m
```
Comment (jhoanmanuelms) [inline gradle.properties:2]: Missing new end of file

**Review by jhoanmanuelms — APPROVED** [review]
LGTM. It looks like most of the changes were just for formatting however, I left a couple of comments, please take a look.

File: commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/security/RegisterRequest.kt (hunk 1 above)
Comment (Juan-EBI) [reply commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/security/RegisterRequest.kt:6]: lets discuss this and address in individual version

**Pull request merged**